CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
```

Comment:
Release note for Version 3.0 that the library now uses semantic versioning and has up to date documentation?